src/components/ui/input.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+import { cn } from "@/lib/utils";
+
+const Input = React.forwardRef<HTMLInputElement, React.ComponentProps<"input">>(
```

Comment:
not blocking this PR but maybe we can look into React 19 and use the `ref` props now that it's officially GA. Maintaining forwardRef for RadixUI can be pretty tedious.